--- CMakeLists.txt.orig	2024-03-04 08:11:52.000000000 +1100
+++ CMakeLists.txt	2024-06-03 10:34:19.470000000 +1000
@@ -94,17 +95,6 @@
   message(FATAL_ERROR "Could not find git")
 endif()
 
-# Find Pandoc
-if(NOT PANDOC_PATH)
-  find_program(PANDOC_PATH NAMES "pandoc" DOC "Pandoc program location")
-
-  if(PANDOC_PATH STREQUAL "PANDOC_PATH-NOTFOUND")
-    message(FATAL_ERROR "Could not find pandoc")
-  else()
-    message(STATUS "Found Pandoc: ${PANDOC_PATH}")
-  endif()
-endif()
-
 # Prevent overriding the parent project's compiler/linker
 # settings on Windows
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
